tests/testsuite/artifact_dep.rs

```diff
@@ -1568,12 +1568,174 @@ fn artifact_dep_target_specified() {
         .with_status(0)
         .run();
 
-    // TODO: This command currently fails due to a bug in cargo but it should be fixed so that it succeeds in the future.
     p.cargo("tree -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
-        .with_stdout_data("")
-        .with_stderr_data(r#"...
-[..]did not find features for (PackageId { name: "bindep", version: "0.0.0", source: "[..]" }, NormalOrDev) within activated_features:[..]
+        .with_stdout_data(str![[r#"
+foo v0.0.0 ([ROOT]/foo)
+└── bindep v0.0.0 ([ROOT]/foo/bindep)
+
+"#]])
+        .with_status(0)
+        .run();
+}
+
+/// From issue #10061
```

Comment:
This should say [#10593 (comment)](https://github.com/rust-lang/cargo/issues/10593#issuecomment-1341214604)